CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ CHANGELOG
 [Next release](https://github.com/rebing/graphql-laravel/compare/4.0.0...master)
 --------------
 
+
+2020-01-19, 4.0.0
+-----------------
```

Comment:
Please remove these lines, as the net release date isn't decided yet!

Reply:
Sure will do. I haven't added this initially, but saw in the PR checklist it was required to amend CHANGELOG.md and wasn't sure what exactly to add. Are you happy to remove the whole entry I've added?

Reply:
No, please keep the changelog (it's a "change log", after all 😜) but only remove the lines I covered in github!

Reply:
@mfn Right, should be ok now I think :)